docs/snapshot_restore.md

```diff
@@ -8,6 +8,10 @@ snapshot and creates the exact same virtual machine, restoring the previously
 saved states. The new virtual machine is restored in a paused state, as it was
 before the snapshot was performed.
 
+## Limitations
+
+VFIO devices and Intel SGX are out of scope. For example, this includes the `--fs` option for `virtio-fs`.
+
```

Comment:
I think it is better to have the limitation section in the end of the documentation. Also, snapshot/restore does not work with `virtio-fs`, mostly because the lack of support from virtiofsd (as you added to the other documentation). We don't need to mentioned it again here. I'd suggest you drop the second commit.

Reply:
My last comment above applies to this as well.